infrastructure/helm-chart/templates/controller/configmap.yaml

```diff
@@ -9,7 +9,7 @@ data:
       "repositories": [
         {
           "name": "airy-core",
-          "url": "https://github.com/airyhq/airy/tree/develop/infrastructure/helm-chart/charts/components/charts"
+          "url": "https://helm.airy.co"
```

Comment:
What if the repository requires username/password? Should we also document that and is it considered in the controller code?